contracts/external/cw-fund-distributor/README.md

```diff
@@ -0,0 +1,37 @@
+# cw-fund-distributor
+
+This contract is meant to facilitate fund distribution 
+proportional to the amount of voting power members have
+at a given block height.
+
+Possible use cases may involve:
+- Dissolving a DAO and distributing its treasury to members prior to shutting down
+- Distributing funds among DAO members
+- Funding subDAOs
+
+## Funding Period
+
+Contract is instantiated with a `funding_period` - a time duration that should suffice 
+to move the funds to be distributed into the distributor contract.
```

Comment:
Why not just instantiate the contract with funds, similar to `cw-vesting`?

Seems like it would be less complicated and easier to setup? https://github.com/DA0-DA0/dao-contracts/pull/608

Reply:
Ah, I see. It makes it so you can send whatever to it to be distributed. Hmmmmmmm...